### PR TITLE
Assignment 2 - Hunter Thompson & Nolan Posey

### PR DESCRIPTION
**Pull request by StrangeRanger**
@Nolan-Posey

Link: https://proud-desert-02a266210.3.azurestaticapps.net/

### CODE REVIEW

File: wordle-web/src/scripts/wordsService.ts [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+        newList.push(element)
+      }
+    })
+
+    return newList
   }
 
   // From: https://github.com/kashapov/react-testing-projects/blob/master/random-word-server/five-letter-words.json
-  static readonly #words: string[] = [
+  private static readonly words: string[] = [
```
Comment (dpeterson8) [inline wordle-web/src/scripts/wordsService.ts:28]: Could have left static and still gotten same results

File: wordle-web/src/views/WordleView.vue [hunk 2]
```diff
@@ -1,23 +1,71 @@
...
-  <h1>Wordle Mind Bender</h1>
+  <main class="align-center">
+    <br />
+    <h1>The Good Word</h1>
+    <br />
 
-  <GameBoard :game="game" @letterClick="addChar" />
+    <div>
+      <GameBoard :game="game" @letterClick="addChar" />
+      <br />
```
Comment (dpeterson8) [inline wordle-web/src/views/WordleView.vue:9]: Could use padding or margin for this possibly, not positive where br should be used.

File: wordle-web/src/components/KeyBoard.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <v-row class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
-    <v-col v-for="(letter, j) in key" :key="j">
+  <v-row rows="auto" class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
```
Comment (dpeterson8) [inline wordle-web/src/components/KeyBoard.vue:2]: I don't believe rows="auto" will change anything after testing on my code as well

**Review by dpeterson8 — COMMENTED** [review]
looks good, site is up and running as well

File: wordle-web/src/scripts/wordsService.ts (hunk 1 above)
Comment (StrangeRanger) [reply wordle-web/src/scripts/wordsService.ts:27]: `static` was never removed. If you're referring to the `#`, I know. I just prefer explicitly saying `private` to utilizing `#` to signify a private variable.

File: wordle-web/src/components/KeyBoard.vue (hunk 3 above)
Comment (Nolan-Posey) [reply wordle-web/src/components/KeyBoard.vue:2]: You're right, I just removed rows="auto" and it didn't change anything. Cols="auto" is the important part to get all the keys lined up together neatly.

File: wordle-web/src/views/WordleView.vue (hunk 2 above)
Comment (Nolan-Posey) [reply wordle-web/src/views/WordleView.vue:9]: I like how the new line (br) adds space between the game board and the components below. We will play around with using padding and/or margins to see what we can do with it.

File: wordle-web/src/views/AboutView.vue [hunk 4]
```diff
@@ -1,17 +1,38 @@
...
+    </v-card>
+
+    <v-card color="cyan-darken-3" class="ma-5" elevation="5" rounded="xl">
+      <v-card-title>Our Teacher Meg</v-card-title>
+      <v-card-text>
+        Meg has been teaching us CSharp and Vuetify since Winter quarter of the 2022-2023 school
+        year. Her ability and passion to teach has helped us hone the necessary skills we as
+        programmers need in order to be successful in our careers. Meg is over-worked and
+        under-paid, but since we have no money... and no influence over her bosses Mark and Grant...
+        we have decided to gift her with this cute image Nolan's cat Nala.
```
Comment (mmwoodfo) [inline wordle-web/src/views/AboutView.vue:28]: My work anniversary is coming up, so I'll be sure to forward this comment about me needing a raise to Mark and Grant. :) 

Here is my cat and a half in return.

image

File: wordle-web/src/components/AppBar.vue [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  <v-app-bar id="toolbar">
+    <v-toolbar-title>
+      <div @click="router.push('/')" id="toolbar-title">
+        <v-icon icon="mdi-book-open-page-variant" />
+        The Good Word
+      </div>
+    </v-toolbar-title>
+
+    <v-btn @click="toggleMusic">
+      <v-icon size="large">{{ isPaused ? 'mdi-volume-high' : 'mdi-volume-mute' }}</v-icon>
```
Comment (mmwoodfo) [inline wordle-web/src/components/AppBar.vue:11]: I think I would switch these icons. Consider how the windows icon is used... it shows as muted when the sound is muted and as on when the sound is on
image

File: wordle-web/src/components/AppBar.vue [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+  theme.global.name.value = 'cowboy'
+  localStorage.setItem('theme', 'cowboy')
+}
+function toggleAquaMode() {
+  theme.global.name.value = 'aqua'
+  localStorage.setItem('theme', 'aqua')
+}
+</script>
+
+<style scoped>
```
Comment (mmwoodfo) [inline wordle-web/src/components/AppBar.vue:152]: You can acheive alot of this without the css class, using padding and margin classes and the height/width attributes. Vuetify also has font classes under their typography docs

File: wordle-web/src/components/AppBar.vue [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+    <v-btn @click="toggleMusic">
+      <v-icon size="large">{{ isPaused ? 'mdi-volume-high' : 'mdi-volume-mute' }}</v-icon>
+    </v-btn>
+
+    <v-btn icon="mdi-cog" @click.stop="settings = !settings"></v-btn>
+
+    <v-app-bar-nav-icon @click.stop="menu = !menu"></v-app-bar-nav-icon>
+  </v-app-bar>
+
+  <v-dialog transition="fab-transition" width="350px" title="Settings" v-model="settings" temporary>
```
Comment (mmwoodfo) [inline wordle-web/src/components/AppBar.vue:19]:
```suggestion
  <v-dialog transition="fab-transition" width="350px" title="Settings" v-model="settings" temporary>
```
I learned something new, I didn't know you could do these transitions on dialogs. :)

**Review by mmwoodfo — COMMENTED** [review]
Great job!

**Pull request closed without merging**